app/controllers/events_controller.rb

```diff
@@ -15,8 +15,9 @@ def schedule
   # GET /events
   # GET /events.json
   def index
-    @events = Event.all
-    @page_title = "All Events"
+    # by default show all scheduled or in-session events
+    @events = params['all_events'] == "true" ? Event.all : Event.where('end_time > ?', Time.now)
+    @page_title = "Scheduled and In-Session Events"
```

Comment:
I don't think this will work with a ternary. Won't this code give the same page title for both conditions ? That is, when I click "All Events" the page title will still be "Scheduled"